relayer/README.md

```diff
@@ -77,13 +77,13 @@ endpoint = "ws://localhost:9545/"
 address = "0x17f7C1e314180D8b8588CA50cF09A0e0847c77F6"
 abi = "/tmp/Bridge.json"
 
-[ethereum.apps.eth]
```

Comment:
was hoping to actually get the substrate changes in this same PR such that we could merge the whole PR in together and keep the main branch fully functional, rather than merging in one piece at a time. anyway either way is fine, i don't really think it makes a major difference.